source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -0,0 +1,159 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Java Driver Version
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+     - MongoDB 3.4
+     - MongoDB 3.2
+     - MongoDB 3.0
+     - MongoDB 2.6
+
+   * - Version 3.12
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.11
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.10
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.9
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.8
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.7
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.6
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.5
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.4
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.3
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.2
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.1
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - Version 3.0
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - Version 2.14
+     -
+     -
+     -
+     -
+     - ✓ [*]_
```

Comment:
This results in a clickable link being inserted. Would `(*)` work?

Reply:
This is meant to be a footnote, but it is not working yet in nextgen. I created a ticket this morning: https://jira.mongodb.org/browse/DOP-931